test/integration/targets/setup_wildfly_server/tasks/main.yml

```
@@ -0,0 +1,75 @@
+- name: Install java
+  package:
+    name: java-1.8.0-openjdk-devel
+
+- name: Download wildfly
+  get_url:
+    url: 'https://download.jboss.org/wildfly/{{ wf_version }}/wildfly-{{ wf_version }}.tar.gz'
+    dest: '/tmp/wildfly-{{ wf_version }}.tar.gz'
```

Use `remote_tmp_dir` for all paths:

```suggestion
    dest: '{{ remote_tmp_dir }}/tmp/wildfly-{{ wf_version }}.tar.gz'
```

The same applies to the other paths below.